### ISSUE TITLE
double click mgdb from VS sometimes opens wrong file

### ISSUE THREAD

**Issue by theZMan** [mono/MonoGame#3230]
Sometimes the GUI tool opens the most recent mgcb not the one you double clicked on

In my case it was a brand new DX Windows project.

**Comment by nkast** [comment]
I found a couple of entries in the registry, one of them missing double quotes around the pipilinepath and a default parameter.

Save the following two lines into a text file pipeline.reg , right click and 'Merge'. That fixed it for now.

[HKEY_CLASSES_ROOT\MonoGame.ContentBuilderFile\shell\open\command]
@="\"C:\\Program Files (x86)\\MSBuild\\MonoGame\\v3.0\\Tools\\Pipeline.exe\" \"%1\""

**Comment by tomspilman** [comment]
@nkast - That bug is coming from here then:

https://github.com/mono/MonoGame/blob/develop/Installers/Windows/MonoGame.nsi#L107

Just need someone to tweak it, build and run the installer to test it.

**Comment by tomspilman** [comment]
Anyone have time to investigate this one?

**Comment by nkast** [comment]
the error must be at line 65, it has to out output `"%1"` at the end of the registry key and possibly wrap the path with quotes as well. But I am not familiar with the sintax of this installer.
https://github.com/mono/MonoGame/blob/develop/Installers/Windows/MonoGame.nsi#L65
the output should look like this 
`"C:\Program Files (x86)\MSBuild\MonoGame\v3.0\Tools\Pipeline.exe" "%1"`

**Comment by nkast** [comment]
I tried NSIS 3.0b1 and 2.46 with the MonoGame.nsi and in both cases I got "!include: could not find: "header.nsh"". I am only guessing here, it's not clear from the teamcity build logs what type of installer the *.nsi is for.

I could add the "%1" and push a branch, wait for teamcity to build and then test the installer, but in case it fails it means more garbage PR. I don't know how .NSI handle quotes,double-quotes, escapes, etc.
